packages/react-scripts/template/public/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "React App",
+  "name": "React App",
```

Comment:
Rather than detail everything in the `create-react-app` docs, I've added an explicit section for "Progressive Web App Metadata" that includes a brief overview and points developers to https://developers.google.com/web/fundamentals/engage-and-retain/web-app-manifest/

That link should remain authoritative.